index.html

```diff
@@ -0,0 +1,175 @@
+                </ul>
+                <button type="submit" class="project">See Project</button>
+            </section>
+        </div>
+    </div>
+    <section class="about" id="about">
+        <div class="about-content1">
+            <h2>About Myself</h2>
+            <p>
+                Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <ul class="social-1">
+                <li><a href="index.html"><img src="image/1.png" alt="Twitter"/></a></li>
+                <li><a href="index.html"><img src="image/2.png" alt="Linkedin"/></a></li>
+                <li><a href="index.html"><img src="image/3.png" alt="Medium"/></a></li>
+                <li><a href="index.html"><img src="image/4.png" alt="Github"/></a></li>
+                <li><a href="index.html"><img src="image/5.png" alt="Angellist"/></a></li>
+            </ul>
+            <button type="submit" class="project">Get my resume</button>
+        </div>
+        <div class="about-content2">
+            <ul class="skills">
+                <li id="list"><a href="index.html">Languages</a></li>
+                <div class="extra">
+                        <a href="index.html">
+                            <img src="image/Ellipse-1.png" alt="JavaScript" />
+                            <p class="list-4">JavaScript</p>
+                        </a>
+                        <a href="index.html">
+                            <img src="image/Ellipse-3.png" alt="HTML" />
+                            <p>HTML</p>
+                        </a>
+                        <a href="index.html">
+                            <img src="image/Ellipse-2.png" alt="Css" />
+                            <p>css</p>
+                        </a>
+                    </li>
+                </div>
+                <li id="list">Frameworks</li>
+                <li id="list">Skills</li>
+            </ul>
+        </div>  <section class="contact-me" id="contact">
+            <h2>Contact me</h2>
+            <p>
+                If you have an application you are interested in developing, 
+                a feature that you need built or a project that needs coding. 
+                i'd love to help with it.</p>
+            <form action="https://formspree.io/f/{form_id}" methop="post" class="text-me">
+                <input type="text" class="simple" placeholder="User name" maxlength="30" required>
+                <input type="email" class="simple" placeholder="Email" required>
+                <textarea class="text-area" placeholder="comment" required></textarea>
+                <button type="submit" class="project">Get in touch</button>
+                </form>      
```

Comment:
- [ ] YOu seem to be having white space around your entire website. Kindly fix it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/213791999-20b436e5-b014-47e2-98f1-6d65e7f4ad53.png)